templates/institutional/includes/inst-info.html

```diff
@@ -0,0 +1,71 @@
+{%- comment -%}
+
+++++++++++++++++++++++++++++++++++
++ List of institutional links and head of institution image
++
++ Used by institutional pages
+++++++++++++++++++++++++++++++++++
+
+{%- endcomment -%}
+
+<section class="gc-crprt">
```

Comment:
Would this code sample be in the "components" under "gc-crprt". It look very similar to the other code sample in organisation template.

I suggest to put both content sample under "components/gc-crprt/samples/" folder